popup.js

```diff
@@ -0,0 +1,322 @@
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['CANOPY', 'Back End Dev', '2015'],
+  },
+  {
+    card: 2,
+    name: 'Multi-Post Stories',
+    name2: 'Multi-Post Stories',
+    image: 'images/story.png',
+    image2: 'images/multi2.png',
+    description: `Experimental content creation feature that allows users to add to
+                              an existing story over the course of a day without spamming their
+                              friends.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['FACEBOOK', 'Full Stack Dev', '2015'],
+  },
+  {
+    card: 3,
+    name: 'Tonic',
+    name2: 'Facebook 360',
+    image: 'images/snap1.png',
+    image2: 'images/portcard.png',
+    description: ` Exploring the future of media in Facebook's first Virtual Reality
+                              app; a place to discover and enjoy 360 photos and videos on Gear
+                              VR.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['FACEBOOK', 'Full Stack Dev', '2015'],
+  },
+  {
+    card: 4,
+    name: 'Multi-Post Stories',
+    name2: 'Uber Navigation',
+    image: 'images/snap3.png',
+    image2: 'images/uber2.png',
+    description: `  A smart assistant to make driving more safe, efficient, and fun by
+                              unlocking your most expensive computer: your car.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['UBER', 'Lead Developer', '2018'],
+  },
+];
+
+const portfolioSection = document.getElementById('portfolio');
+portfolioSection.innerHTML = projectCards
+  .map(
+    (project, index) => `
+  <section class="card ${index % 2 !== 0 ? 'image-left' : 'image-right'}">
+<div class="content-right">
+          <img class="mobileCard" src="${project.image}" alt="Tonic image" />
+          <img
+            class="desktopCard"
+            src="${project.image2}"
+            alt="Nature image"
+          /></div>
+
+          <div class="content-left">
+          <h2 class="mobileheadline">${project.name}</h2>
+          <div class="">
+            <h2 class="desktopheadline">${project.name2}</h2>
+            <div id="profs" >
+              <h3 id="cannopy">${project.specifics[0]}</h3>
+              <img id="bullet" src="images/bullet.png" alt="" />
+              <h3 class="profList">${project.specifics[1]}</h3>
+              <img  id="bullet2"src="images/bullet.png" alt="" />
+              <h3 class="profList">${project.specifics[2]}</h3>
+            </div>
+            <div id="prof2">
+              <h3  id="cannopy">${project.specifics2[0]}</h3>
+              <img src="images/bullet.png" alt="Counter icon" />
+              <h3 class="">${project.specifics2[1]}</h3>
```

Comment:
- [ ] You have implemented the functionality well but your styling is not proper as the Figma design. I would suggest you stick as much as possible to Figma's design. You have spacing on the left and right and top and button, If you see on Figma design you will see no spacing. Kindly implement it.
## your design
![your-design](https://user-images.githubusercontent.com/81727988/162563007-f0c252f1-91cd-487d-a7f1-15bd58b238e4.png)
## Figma design
![figma-design](https://user-images.githubusercontent.com/81727988/162563016-01b7c48e-8813-4fd0-8c76-f13a747041d5.png)